app/controllers/api/internal/v2/court_applications_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Api
+  module Internal
+    module V2
+      class CourtApplicationsController < ApplicationController
+        def show
+          # TODO: this will be the new application details endpoint
+          raise NotImplementedError
+        end
+      end
+    end
+  end
+end
```

Comment:
this is a placeholder for the action for the `internal/court_applications/id`  endpoint.